spel/minimal-linux.json

```
@@ -60,6 +60,39 @@
             "type": "amazon-ebs",
             "user_data_file": "{{ template_dir }}/userdata/tmpfsroot-el7.cloud"
         },
+        {
+            "ami_description": "STIG-partitioned [*NOT HARDENED*], LVM-enabled, XFS-formatted, \"minimal\" CentOS 7 AMI, with updates through {{ isotime \"2006-01-02\" }}. Default username `maintuser`. See {{ user `spel_desc_url` }}.",
+            "ami_groups": "{{ user `ami_groups` }}",
+            "ami_name": "{{ user `spel_identifier` }}-minimal-centos-7-hvm-{{ user `spel_version` }}.x86_64-gp2-xfs",
```

To avoid the hard-coding, maybe try using `{{ build_name }}` here in the ami name value...

```suggestion
            "ami_name": "{{ user `spel_identifier` }}-{{ build_name }}-{{ user `spel_version` }}.x86_64-gp2",
```

---

As I put in Note<sup>1</sup> of my opening PR comment, `{{ build_name }}` isn't suitable. It results in an `ami_name` value of `spel-minimal-rhel-7-hvm-xfs-2019.05.01-rc2.x86_64-gp2-xfs` or `spel-minimal-centos-7-hvm-xfs-2019.05.01-rc2.x86_64-gp2-xfs`. That said, there was a `sed` function in the documentation. Changing `ami_name` to:

~~~
{{ user `spel_identifier` }}-{{ sed \"s/-xfs//\" build_name }}-{{ user `spel_version` }}.x86_64-gp2-xfs"
~~~

Produces:

~~~
$ bash ~/buildit.sh
minimal-centos-7-hvm-xfs output will be in this color.
minimal-rhel-7-hvm-xfs output will be in this color.

==> minimal-centos-7-hvm-xfs: Prevalidating AMI Name: spel-minimal-centos-7-hvm-2019.05.01-rc2.x86_64-gp2-xfs
==> minimal-rhel-7-hvm-xfs: Prevalidating AMI Name: spel-minimal-rhel-7-hvm-2019.05.01-rc2.x86_64-gp2-xfs
~~~

---

Just remove `xfs` from the end. Doesn't really matter where the token ends up

---

Euw.

---

Well, if you keep it at the end, you'll also need to think through how to present or update the console search links in the readme...